docs/source/examples/substrafl/go_further/run_mnist_cyclic.ipynb

```diff
@@ -254,16 +254,16 @@
         "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from typing import Any, List, Optional, Union, Dict, Callable
</pre><p>why are we reverting from the isort recommended syntax?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Not intended, sorry